third_party/CMakeLists.txt

```diff
@@ -3,7 +3,7 @@
 # SPDX-License-Identifier: Apache-2.0
 #
 
-set(TT_MLIR_VERSION "5fd26305487fd4b28eabd3f5de826f35d65b09d0")
+set(TT_MLIR_VERSION "41a087ae44c115ad5bfe06b7689dff84bbaf99a5")
```

Comment:
[Last uplift ](https://github.com/tenstorrent/tt-xla/pull/74)passed pointing to `a0c039c79ce8481fc94aa804c73b66ae8558d66f ` which is newer than this one.